lib/rubocop-rails.rb

```diff
@@ -3,6 +3,7 @@
 require 'rubocop'
 require 'rack/utils'
 require 'active_support/inflector'
+String.remove_method(:blank?) if String.instance_method(:blank?)
 require 'active_support/core_ext/object/blank'
```

Comment:
(This require statement is slightly misleading. It refers to `active_support/core_ext/object/blank`, but it not only patches `Object`, but also several other classes, [including `String`](https://github.com/rails/rails/blob/v8.0.1/activesupport/lib/active_support/core_ext/object/blank.rb#L135-L153).)